internal/pkg/template/templates/workloads/partials/cf/rollback-alarms.yml

```diff
@@ -0,0 +1,51 @@
+{{- if .DeploymentConfiguration.Rollback.CPUUtilization}}
+CPURollbackAlarm:
+  Metadata:
+    'aws:copilot:description': "A CloudWatch alarm associated with CPU utilization for deployment rollbacks"
+  Type: AWS::CloudWatch::Alarm
+  Properties:
+    AlarmDescription: "Roll back ECS service if CPU utilization is greater than or equal to {{.DeploymentConfiguration.Rollback.CPUUtilization}}% twice in 2 minutes."
+    AlarmName: {{.DeploymentConfiguration.Rollback.TruncateAlarmName .AppName .EnvName .WorkloadName "CopilotRollbackCPUAlarm"}}
+    Namespace: 'AWS/ECS'
+    Dimensions:
+      - Name: ClusterName
+        Value:
+          Fn::ImportValue:
+            !Sub '${AppName}-${EnvName}-ClusterId'
+      - Name: ServiceName
+        Value: !Select [2, !Split ["/", !Ref Service]]
```

Comment:
Just to make sure the CFN stack did deploy right with `rollback_alarms` for a brand new LBWS?
I wanted to make sure that CFN created the alarms _after_ the service creation so the `DependsOn` is not needed

Reply:
Yup yup!